java-manta-client-unshaded/src/main/java/com/joyent/manta/config/ConfigContext.java

```diff
@@ -362,6 +368,10 @@ static void validate(final ConfigContext config) {
             encryptionSettings(config, failureMessages);
         }
 
+        if (BooleanUtils.isTrue(config.isContentTypeDetectionEnabled())) {
+            encryptionSettings(config, failureMessages);
```

Comment:
Why are you changing encryption settings here?

Reply:
Added erroneously, removed .